From 7900e4921a923a5f9c6105b71d824164825147b1 Mon Sep 17 00:00:00 2001
From: navinashok swaminathan <navinashok.sg@gmail.com>
Date: Mon, 6 Sep 2021 06:19:54 +0000
Subject: [PATCH] Create Patch File

---
 client3.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client3.py b/client3.py
index 9c943dc..15e92c9 100644
--- a/client3.py
+++ b/client3.py
@@ -1,4 +1,4 @@
-################################################################################
+#################################################################################
 #
 #  Permission is hereby granted, free of charge, to any person obtaining a
 #  copy of this software and associated documentation files (the "Software"),
-- 
2.17.1

